### PR TITLE
String translations for x.46, and add Albanian and Korean

### PR DESCRIPTION
**Pull request by mazameli**
This PR is always hard to review. You're mostly looking out to make sure that I'm not touching files that I shouldn't: this should only affect language files in `/locales/`

This should backport to the 46 release branch.

### CODE REVIEW

**Comment by mazameli** [comment]
@alxnddr @nemanjaglumac, hold off on merging or approving this actually — I'm going to push another commit later today with some last-minute translation additions. [Context here](https://metaboat.slack.com/archives/C01MS7DQKR6/p1678975171971489?thread_ts=1678906612.528139&cid=C01MS7DQKR6).

**Comment by mazameli** [comment]
@alxnddr @nemanjaglumac, okay, ready for "review" and approval! 🙏🏼

**Comment by mazameli** [comment]
I'm trying to resolve this merge conflict by merging in master, but every time I try to complete the merge I'm getting `husky - pre-commit hook exited with code 1 (error)`

File: .gitignore
Comment (mazameli) [inline .gitignore:88]: This showed up after I merged master — not sure how. What should I do to resolve this?

Comment (iethree) [reply .gitignore:88]: `git rebase master`

**Review by alxnddr — APPROVED** [review]

**Review by alxnddr — APPROVED** [review]

**Comment by mazameli** [comment]
@metabase-bot backport release-x.46.x

**Pull request merged**
